vulcanai/plotters/visualization.py

```diff
@@ -401,7 +401,7 @@ def display_receptive_fields(network, top_k=5, save_path=None):
     fig.suptitle("Feature importance")
     num_layers = len(network._modules['network'])
     for i, layer in enumerate(network._modules['network']):
-        raw_field = layer._kernel._parameters['weight'].detach().numpy()
+        raw_field = layer._kernel._parameters['weight'].detach().cpu().numpy()
```

Comment:
could you put the `.cpu()` before the `.detach()`?

Reply:
👍